### DISCUSSION TITLE
Capacitive stylus issues

### DISCUSSION THREAD

**Discussion by ciano1999** [opengisch/QField#5328] [Bugs]
Since the latest QField update the capacitive stylus is only working sporadically when trying to draw features. The stylus works fine until I use my fingers to pan or zoom, then the cursor 'locks' to the centre of the screen and will no longer track when I hover the stylus over the screen.

I am using a Galaxy Active Tab Pro.

Has anyone else had this issue?

**Comment by lp-dj** [comment]
Since I started using QField (2019/20?), the user experience with stylus usage has been an up and down one. There have been a number of regarding issues (e.g.  #5010) that have since been closed. That has improved things recently. However, I still (or again?) have problems, too.

I also use the Galaxy Tab Active Pro. It has to be said that the device has unfortunately reached its EOL and has not received any updates for a year now. And in the end, these were only security updates. The last major update was over 3 years ago! (https://doc.samsungmobile.com/SM-T545/XEH/doc.html)

I hate to say it because the device is still in top condition on the hardware side: But maybe it's time for an update? Probably on the hardware side, as I couldn't find a (current) custom rom for the tablet to give it an up-to-date operating system.

I would be interested to know whether the old hard- and software could also be contributing to the problems with stylus use, for example because the required libraries are not up to date? Anyone here with a current device/operating system who still has problems using the Stylus?
